DiamondLightSource/hyperion
# Use new OAV parameter definition

Follows changes in the OAVParameters in dodal.

Link to dodal PR (if required): [#239](https://github.com/DiamondLightSource/dodal/pull/239)


### To test:
1. Check that the changes haven't affected the plans
3. Run tests
